src/erk/cli/commands/pr/land_cmd.py

```diff
@@ -265,6 +276,17 @@ def pr_land(
         activate_worktree(ctx, post_deletion_repo, target_path, script, command_name="pr-land")
         # activate_worktree raises SystemExit(0)
     else:
+        # Construct git pull commands if pull_flag is set
+        post_commands: list[str] | None = None
+        if pull_flag:
+            trunk_branch = ctx.git.detect_trunk_branch(main_repo_root)
+            post_commands = [
+                f'__erk_log "->" "git pull origin {trunk_branch}"',
```

Comment:
is __erk_log command in the shell activation script?

Reply:
Resolved via /erk:pr-address at 2025-12-30 06:43 AM PST - yes, __erk_log is defined in _render_logging_helper() which is included in every activation script

_Addressed via `/erk:pr-address` at 2025-12-30 06:43 PST_